modules/test/playwright/tests/portal-workflow-task-web/workload.spec.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {notificationPagesTest} from '../../fixtures/notificationPagesTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import clearUserNotifications from '../../utils/clearUserNotifications';
+import retryGetWorkflowTasksBySubmittingUser from '../../utils/retryGetWorkflowTasksBySubmittingUser';
+import {blogsPagesTest} from '../blogs-web/fixtures/blogsPagesTest';
+
+export const test = mergeTests(
+	applicationsMenuPageTest,
+	apiHelpersTest,
+	blogsPagesTest,
+	loginTest(),
+	notificationPagesTest,
+	workflowPagesTest
+);
+
+interface CreatedEntities {
+	blogPosts?: TBlogPost[];
+	users?: TUserAccount[];
+}
+
+const createdEntities: CreatedEntities = {};
+
+test.afterEach(async ({apiHelpers, configurationTabPage, page}) => {
+	await configurationTabPage.goTo();
+
+	await configurationTabPage.unassignWorkflowFromAssetType('Blogs Entry');
+
+	if (createdEntities.users?.length) {
+		for (const user of createdEntities.users) {
+			await apiHelpers.headlessAdminUser.deleteUserAccount(
+				Number(user.id)
+			);
+		}
+	}
+
+	delete createdEntities.users;
+
+	if (createdEntities.blogPosts?.length) {
+		for (const blog of createdEntities.blogPosts) {
+			await apiHelpers.headlessDelivery.deleteBlog(blog.id);
+		}
+	}
+
+	delete createdEntities.blogPosts;
+
+	await clearUserNotifications({
+		notificationText: 'Notification no longer applies.',
+		page,
+	});
+});
+
+test('view workload distribution for all assignees', async ({
+	allItemsPage,
+	apiHelpers,
+	applicationsMenuPage,
+	configurationTabPage,
+	metricsPage,
+	page,
+}) => {
+	const numberOfUsersAndTasks = 5;
+	const newPageSize = 4;
+	const workflowDefinitionName = 'Single Approver';
```

Comment:
Hi @cgmonte, when I see these values ​​hardcoded defined by us. I wondered if we need to use the constant variables pattern, like:

`cosnt NUMBER_OF_USERS_AND_TASKS = 5`

I would like to know what you think